psi4/driver/p4util/python_helpers.py

```diff
@@ -442,6 +442,9 @@ def set_options(options_dict, verbose=1):
     rejected = {}
 
     for k, v, in options_dict.items():
+
+        k = k.strip()
```

Comment:
neat, maybe add to following line `optionre.match(k.strip())` so avert the reference/value issue FAE brought up.

Reply:
Thanks!